src/imgtools/cli/query.py

```
@@ -0,0 +1,112 @@
+        logger.warning(f"n_jobs requires a directory as input. {path} will be used as the index.")
+
+    elif (path.is_dir()):
+        from imgtools.dicom.crawl import Crawler
+        logger.info("Initializing crawl.")
+        try:
+            crawler = Crawler(
+                dicom_dir=path,
+                n_jobs=n_jobs,
+                force=force,
+            )
+            crawl_result = crawler.crawl()
```

Pylint (3.3.7)

[error] Assigning result of a function call, where the function has no return

(E1111)